ultrasonic/src/main/kotlin/org/moire/ultrasonic/service/MediaPlayerService.kt

```diff
@@ -538,7 +552,8 @@ class MediaPlayerService : Service() {
             }
         }
 
-        playbackState.setState(state, PlaybackStateCompat.PLAYBACK_POSITION_UNKNOWN, 1.0f)
+        // TODO playerPosition should be updated more frequently (currently this function is called only when the playing track changes)
```

Comment:
This is necessary for the Android Auto to show playback position in the track during playback.